### DISCUSSION TITLE
make custom border for each panel available, is it possible?

### DISCUSSION THREAD

**Discussion by TBEvil** [grafana/grafana#38870] [Dashboards, Panels & Visualizations]
1 users can upload his own borders
2 users can set border for each panel

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
